docs/api-documentation.md

```diff
@@ -1,15 +1,20 @@
 # API documentation
 
+:::info
+
+Airbyte Cloud does not currently support API access yet. Our Cloud API will be available in 2023. For early-access in 2023, please email riley@airbyte.io
```

Comment:
Sharing future announcements in Docs is usually a bad practice. Happy to chat if you think it's worth making an exception for the Cloud API.

Reply:
Thats fair - just updating page to reflect the current state of things.

• Config API will not be receiving "improvements over next couple weeks"
• Cloud does/will support API access.

I can update this to "Airbyte Cloud will support API access in 2023. See [Airbyte roadmap](https://app.harvestr.io/roadmap/view/pQU6gdCyc/airbyte-roadmap) for more details.

Unless you have a different approach in mind?

Reply:
Sounds good!

Reply:
Update made. Let me know if there are other reviewers required for this PR.